tradingdb/relationaldb/tests/utils.py

```diff
@@ -0,0 +1,5 @@
: false,       "stateMutability": "view",       "type": "function"     },     {       "constant": true,       "inputs": [         {           "name": "",           "type": "address"         }       ],       "name": "whitelist",       "outputs": [         {           "name": "",           "type": "bool"         }       ],       "payable": false,       "stateMutability": "view",       "type": "function"     },     {       "constant": false,       "inputs": [         {           "name": "to",           "type": "address"         },         {           "name": "value",           "type": "uint256"         }       ],       "name": "transfer",       "outputs": [         {           "name": "",           "type": "bool"         }       ],       "payable": false,       "stateMutability": "nonpayable",       "type": "function"     },     {       "constant": false,       "inputs": [         {           "name": "_admins",           "type": "address[]"         }       ],       "name": "removeAdmin",       "outputs": [],       "payable": false,       "stateMutability": "nonpayable",       "type": "function"     },     {       "constant": false,       "inputs": [         {           "name": "disallowed",           "type": "address[]"         }       ],       "name": "disallowTransfers",       "outputs": [],       "payable": false,       "stateMutability": "nonpayable",       "type": "function"     },     {       "constant": true,       "inputs": [         {           "name": "owner",           "type": "address"         },         {           "name": "spender",           "type": "address"         }       ],       "name": "allowance",       "outputs": [         {           "name": "",           "type": "uint256"         }       ],       "payable": false,       "stateMutability": "view",       "type": "function"     },     {       "anonymous": false,       "inputs": [         {           "indexed": true,           "name": "owner",           "type": "address"         },         {           "indexed": false,           "name": "amount",           "type": "uint256"         }       ],       "name": "Issuance",       "type": "event"     },     {       "anonymous": false,       "inputs": [         {           "indexed": true,           "name": "from",           "type": "address"         },         {           "indexed": true,           "name": "to",           "type": "address"         },         {           "indexed": false,           "name": "value",           "type": "uint256"         }       ],       "name": "Transfer",       "type": "event"     },     {       "anonymous": false,       "inputs": [         {           "indexed": true,           "name": "owner",           "type": "address"         },         {           "indexed": true,           "name": "spender",           "type": "address"         },         {           "indexed": false,           "name": "value",           "type": "uint256"         }       ],       "name": "Approval",       "type": "event"     }   ]')
```

Comment:
maybe this can be taken from chain_events folder?

Reply:
https://github.com/gnosis/pm-trading-db/blob/master/tradingdb/chainevents/abis/TournamentToken.json